lang/java/reef-runtime-azbatch/src/main/java/org/apache/reef/runime/azbatch/driver/REEFEventHandlers.java

```diff
@@ -96,4 +109,8 @@ void onResourceStatus(final ResourceStatusEvent resourceStatusEvent) {
   public void close() throws Exception {
     // Empty, but here for a future where we need to close a threadpool
   }
+
+  public void scheduleAlarm() {
```

Comment:
I think a better place for this would be in AzureBatfchResourceManagerStartHandler. 
Let's keep this class for purely AzureBatchRuntime -> REEF Common interface needs.

Reply:
That's a good idea. Trying it out now.

Reply:
Moved this to the TaskStatusAlarmHandler instead and disabled or enabled the alarm based on whether or not there are containers that are in use.